tests/test_username_api.py

```diff
@@ -40,306 +41,62 @@ def get_expected_response(website, user, status):
               else None
   }
 
+def load_test_cases(type):
+  res = []
+  for website in data:
+    if type == 'available_username':
+      usernames = data[website]['available_usernames']
+    if type == 'taken_username':
+      usernames = data[website]['taken_usernames']
+    if type == 'format_checking':
```

Comment:
Better to break this into it's separate function.

Reply:
umm... breaking into `format_checking` and `taken/available_username`?

Reply:
Indeed, `load_format_checking_cases` and `load_general_test_cases`(or something more descriptive), would be clearer, since format checking is most independent of taken/not taken usernames

Reply:
alright